opensrp-chw-core/src/test/java/org/smartregister/chw/core/fragment/CoreBirthNotificationFragmentTest.java

```diff
@@ -0,0 +1,328 @@
+        Intent intent = shadowOf(childRegisterActivity).getNextStartedActivity();
+        assertNotNull(intent);
+    }
+
+    @Test
+    public void testShowNotFoundPopupShowsDialog() {
+        when(coreFpRegisterFragment.getActivity()).thenReturn(coreChildRegisterActivity);
+        when(coreChildRegisterActivity.getFragmentManager()).thenReturn(fragmentManager);
+        when(fragmentManager.beginTransaction()).thenReturn(fragmentTransaction);
+        coreFpRegisterFragment.showNotFoundPopup("1234");
+        verify(fragmentManager).beginTransaction();
+        verify(fragmentTransaction).addToBackStack(null);
+        verify(fragmentTransaction).add(any(NoMatchDialogFragment.class), eq(DIALOG_TAG));
+    }
+
+    @Test
+    public void getDefaultSortQueryCallsPresenterGetSortQuery() {
+        coreFpRegisterFragment.getDefaultSortQuery();
+        verify(coreFpRegisterFragment.presenter(), Mockito.times(1)).getDefaultSortQuery();
+    }
+
+    @Test
+    public void getDueFilterConditionCallsPresenterGetSortQuery() {
+        coreFpRegisterFragment.getDueFilterCondition();
+        verify(coreFpRegisterFragment.presenter(), Mockito.times(1)).getDueFilterCondition();
+    }
+
+    @Test
+    public void testOnViewClickedDoesNothing() {
+        when(coreFpRegisterFragment.getActivity()).thenReturn(null);
+        coreFpRegisterFragment.onViewClicked(view);
+        verifyZeroInteractions(view);
+    }
+
+    @Test
+    public void testRefreshSyncProgressSpinner() {
+        Whitebox.setInternalState(coreFpRegisterFragment, "syncProgressBar", syncProgressBar);
+        Whitebox.setInternalState(coreFpRegisterFragment, "syncButton", syncButton);
+        coreFpRegisterFragment.refreshSyncProgressSpinner();
+        verify(syncButton).setVisibility(View.GONE);
+    }
+
+    @Test
+    public void refreshSyncProgressSpinnerTogglesSyncVisibility() {
+        ReflectionHelpers.setField(coreFpRegisterFragment, "syncButton", syncButton);
+        ReflectionHelpers.setField(coreFpRegisterFragment, "syncProgressBar", syncProgressBar);
+        coreFpRegisterFragment.refreshSyncProgressSpinner();
+        verify(syncProgressBar, Mockito.times(1)).setVisibility(View.GONE);
+        verify(syncButton, Mockito.times(1)).setVisibility(View.GONE);
+    }
+
+    @Test
+    public void testGetToolBarTitle() {
+        int title = R.string.child_register_title;
+        assertEquals(coreFpRegisterFragment.getToolBarTitle(), title);
+    }
+
+    @Test
+    public void testGetMainCondition() {
+        assertEquals(coreFpRegisterFragment.getMainCondition(), presenter.getMainCondition());
+    }
+
+    @Test
+    public void testGetDefaultSortQuery() {
+        assertEquals(coreFpRegisterFragment.getDefaultSortQuery(), presenter.getDefaultSortQuery());
+    }
+
+    private void getDueFilterCondition() {
+        Mockito.doReturn("").when(coreFpRegisterFragment).getDueFilterCondition();
+    }
+
+    @Test
+    public void testOnResumption() {
```

Comment:
Kindly follow the test naming convention `test[MethodName][ExpectedResult][Parameters]`